plugin/storage/integration/fixtures/queries.json

```diff
@@ -81,8 +81,8 @@
       "ServiceName": "query05-service",
       "OperationName": "",
       "Tags": null,
-      "StartTimeMin": "2017-01-26T15:46:31.639875Z",
-      "StartTimeMax": "2017-01-26T17:46:31.639875Z",
+      "StartTimeMin": "2017-01-26T00:00:31.639875Z",
```

Comment:
so the spans in the fixture were too far apart (i.e. ~ a day) so the reader wasn't able to retrieve it, because i made the logic in the reader based on that traces won't be more than ~2 hours long (idk if you remember talking about this with me a few weeks back)